README.md

```diff
@@ -58,32 +58,32 @@ plugins: [
 
       // If you need to proxy events through a custom data plane,
       // add a `dataPlaneUrl` property (defaults to https://hosted.rudderlabs.com )
-      // RudderStack docs:
-      //   - https://docs.rudderstack.com/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk#3-1-load
+      // Rudderstack docs:
```

Comment:
Replace all the instances of `Rudderstack` with `RudderStack`.

Reply:
Done